recipe/meta.yaml

```diff
@@ -51,17 +55,68 @@ outputs:
         - if not exist %LIBRARY_PREFIX%\\bin\\dnnl.dll exit 1  # [win]
         - if not exist %LIBRARY_PREFIX%\\lib\\dnnl.lib exit 1  # [win]
         - if not exist %LIBRARY_PREFIX%\\include\\dnnl.h exit 1  # [win]
-  - name: {{ name|lower }}-cpu-{{ dnnl_cpu_runtime }}
+
+  - name: onednn-cpu-{{ dnnl_cpu_runtime }}-gpu-{{ dnnl_gpu_runtime }}
+    version: {{ version }}
+    build:
+      string: {{ dnnl_cpu_runtime }}_{{ dnnl_gpu_runtime }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      run_exports:
+        - {{ pin_subpackage("onednn", max_pin="x") }}
+    requirements:
+      host:
+        - {{ pin_subpackage('onednn', exact=True) }}
+      run:
+        - {{ pin_subpackage('onednn', exact=True) }}
+    about:
+      home: https://github.com/oneapi-src/oneDNN
+      license: Apache-2.0
+      license_file: LICENSE
+      summary: oneAPI Deep Neural Network Library (oneDNN)
+    test:
+      commands:
+        - exit 0
```

Comment:
On second thought, this is not necessary. We can keep it if you prefer, but IMO we don't need it.

Reply:
What part is not necessary exactly? The idea was to include both CPU and GPU runtimes in the name of packages to have `onednn-cpu-tbb-gpu-dpcpp`, `onednn-cpu-omp-gpu-dpcpp`, `onednn-cpu-dpcpp-gpu-dpcpp` and `onednn-cpu-threadpool-gpu-dpcpp`. Actually the only one we really need is `onednn-cpu-dpcpp-gpu-dpcpp` which has the same programming model for both CPU and GPU engines, the rest configurations with CPU runtimes (with DPCPP GPU runtime) are optional

Reply:
Right. My thought was that that could be achieved with
```
conda install onednn-cpu-dpcpp onednn-gpu-dpcpp
```
to achieve the same thing. If that's too verbose we can have the package name with `onednn-cpu-dpcpp-gpu-dpcpp`.

Reply:
I see, yeah, let's leave it as `onednn-cpu-dpcpp-gpu-dpcpp`

Reply:
I'd suggest we stick with something like `onednn-sycl`.

Reply:
> I'd suggest we stick with something like `onednn-sycl`.

Will it include both CPU sycl and GPU sycl runtimes? My idea was to keep both runtimes in the name so user can select between `onednn-cpu-tbb-gpu-dpcpp`, `onednn-cpu-omp-gpu-dpcpp`, `onednn-cpu-threadpool-gpu-dpcpp`

Reply:
Right, I was thinking about a single SYCL package with both runtimes. Having binary packages for all possible combinations is a bit overwhelming and will likely cause confusion.

Reply:
ok, sounds good